### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by kohidave**
Update the readme to point to the new binaries and include a section about running one off tasks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ We're distributing binaries from our GitHub releases. Instructions for installin
 
 | Platform | Command to install |
 |---------|---------
-| macOS | `curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/download/v0.2.0/copilot-darwin-v0.2.0 && chmod +x /usr/local/bin/copilot && copilot --help` |
-| Linux | `curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/download/v0.2.0/copilot-linux-v0.2.0 && chmod +x /usr/local/bin/copilot && copilot --help` |
+| macOS | `curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/download/v0.2.0/copilot-darwin-v0.3.0 && chmod +x /usr/local/bin/copilot && copilot --help` |
+| Linux | `curl -Lo /usr/local/bin/copilot https://github.com/aws/copilot-cli/releases/download/v0.2.0/copilot-linux-v0.3.0 && chmod +x /usr/local/bin/copilot && copilot --help` |
```
Comment (efekarakus) [inline README.md]: I just merged the other PR, I don't know why mergify didn't auto-merge. So we can add only the task run section.

File: README.md [hunk 2]
```diff
@@ -68,6 +68,18 @@ This will create a VPC, Application Load Balancer, an Amazon ECS Service with th
 
 Once you're finished playing around with this project, you can delete it and all the AWS resources associated it by running `copilot app delete`.
 
+## Running a one off task 🏃‍♀️
```
Comment (huanjani) [inline README.md]: Should we change this to "one-off" with a hyphen?
(Love the runner.)

File: README.md [hunk 3]
```diff
@@ -68,6 +68,18 @@ This will create a VPC, Application Load Balancer, an Amazon ECS Service with th
...
+
+Copilot makes it easy to set up production ready applications, but you can also use it to run tasks on AWS Fargate without having to set up any infrastructure. To run a simple hello world task on Fargate you can use Copilot:
```
Comment (huanjani) [inline README.md]: Another hyphen here: "production-ready"?

Comment (kohidave) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (kohidave) [reply README.md]: Done!! Thank you!

File: README.md (hunk 1 above)
Comment (kohidave) [reply README.md]: The rebase should merge em!

Comment (kohidave) [reply README.md]: Also I thought you did! haha

**Pull request merged**